test/blackbox-tests/test-cases/include-qualified/basic.t/run.t

```diff
@@ -1,8 +1,19 @@
 Basic test showcasing the feature. Every directory creates a new level of aliasing.
   $ dune build
-  File "lib/dune", line 1, characters 17-26:
-  1 | (include_subdirs qualified)
-                       ^^^^^^^^^
-  Error: Unknown value qualified
-  Hint: did you mean unqualified?
+  File "lib/foolib.ml-gen", line 4, characters 10-11:
+  4 | module Foo.Bar = Foolib__Bar
+                ^
+  Error: Syntax error
   [1]
+
+  $ cat _build/default/lib/foolib.ml-gen 
+  (* generated by dune *)
+  
+  (** @canonical Foolib.Foo.Bar *)
+  module Foo.Bar = Foolib__Bar
```

Comment:
does the value here need to be `Foolib__Foo__Bar`? I think it could conflict with `bar.ml` modules in different subdirs (which could be allowed under `qualified`.

Reply:
This is an artifact from an old test. Not working yet.